src/style-spec/bin/gl-style-composite.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+/* eslint-disable import/no-unresolved, no-process-exit */
+
+import fs from 'fs';
+import minimist from 'minimist';
+import {format, composite} from '../dist/index.es.js';
```

Comment:
Can we rely on the `exports` configuration in `package.json` rather than specifying the exact bundle path?

Reply:
It seems that directory import is not supported when resolving ES modules. It fails with `Error [ERR_UNSUPPORTED_DIR_IMPORT]` when using the `..` here.

But we can import `@mapbox/mapbox-gl-style-spec` itself here, and it seems to work.